services/order.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+
+from django.db import transaction
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    user = User.objects.get(username=username)
+    with transaction.atomic():
```

Comment:
There is no need to have `transaction.atomic` decorator and context manager at the same time, leave only the decorator.